### PR TITLE
Plumb the bug ID through to more of the log messages

### PR DESCRIPTION
**Pull request by andrewpollock**
Because the worker is threaded the log entries don't readily correlate with the bug ID logged earlier in the process, which makes manually following the processing of a bug in the logs more difficult.

### CODE REVIEW

File: osv/impact.py [hunk 1]
```diff
@@ -264,13 +269,14 @@ def _get_commit_to_tag_mappings(repo: pygit2.Repository):
...
 
 
-def get_commit_and_tag_list(repo,
-                            start_commit,
-                            end_commit,
-                            commits_to_tags=None,
-                            include_start=False,
-                            include_end=True,
-                            limit_commit=None):
+def get_commit_and_tag_list(bug_id: str,
```
Comment (oliverchang) [inline osv/impact.py:272]: This is technically a public interface as part of the Python OSV library that we can't change.

File: osv/impact.py [hunk 2]
```diff
@@ -264,13 +269,14 @@ def _get_commit_to_tag_mappings(repo: pygit2.Repository):
...
+                            repo: pygit2.Repository,
+                            start_commit: str,
+                            end_commit: str,
+                            commits_to_tags: dict = None,
+                            include_start: bool = False,
+                            include_end: bool = True,
+                            limit_commit: dict = None):
```
Comment (oliverchang) [inline osv/impact.py:279]: Is dict the correct type here? I think it should be str ?

**Review by oliverchang — COMMENTED** [review]
Hmm, there's a fair bit of plumbing here and a breaking interface change. Could we instead plumb it through via https://github.com/google/osv.dev/blob/abde45effd50e16ceef8039d3fbb14dcfa837716/docker/worker/worker.py#L91 which keeps it consistent with how we plumb through metadata like these generally?

**Comment by andrewpollock** [comment]
Abandoning this approach in favour of https://github.com/google/osv.dev/pull/1430

**Pull request closed without merging**